src/Symfony/Component/PropertyInfo/composer.json

```diff
@@ -28,16 +28,16 @@
     "require-dev": {
         "symfony/serializer": "~2.8|~3.0",
         "symfony/cache": "~3.1",
-        "phpdocumentor/reflection": "^1.0.7",
+        "phpdocumentor/reflection-docblock": "^3.0",
         "doctrine/annotations": "~1.0"
     },
     "conflict": {
-        "phpdocumentor/reflection": "<1.0.7"
+        "phpdocumentor/reflection-docblock": "<3.0"
```

Comment:
This is also if someone want to get the library (as the suggest propose), composer can know which version works with the PropertyInfo component